marketplace/collect.schema.json

```diff
@@ -85,6 +85,26 @@
                             "required": ["join", "on", "url"]
                         }
                     }
+                },
+                "extraOperations": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "mobile.de": {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "enabled": {
+                                    "type": "boolean",
+                                    "default": false
+                                },
+                                "archive": {
```

Comment:
What does this do? Is it really configurable? I thought that there was nothing to configure inside the mobile.de operation.

Reply:
It's the archive url actually. For a partner it can be `ftp://login/Bierschneider.zip` for exemple. It's configurable, not the same for everyone.

Reply:
I don't understand. I thought that the archive is the file that is already defined in the attribute `url`

Reply:
In that case, you have to change the UrlCollector. The configuration files are meant to be used by the marketplace operators, they are not ours. You cannot use them to pass variables from one component to the other.

Reply:
I think It'll be done in that issue https://redmine.aramisauto.com/issues/50083 . No ?

Reply:
I am talking about the configuration here. The configuration requires 1 input file, passed into the `url` parameter. Here you are introducing a second parameter for the same thing, without any good reason. Let's keep the configuration interface clean and we will see what's the correct implementation in a separate ticket.

Reply:
I revert the last change